bin/alluxio

```diff
@@ -352,6 +353,12 @@ function main {
   "clearCache")
     sync; echo 3 > /proc/sys/vm/drop_caches ;
   ;;
+  "metricsDocGen")
+    CLASS="alluxio.cli.MetricsDocGenerator"
+    CLASSPATH=${ALLUXIO_CLIENT_CLASSPATH}
+    ALLUXIO_SHELL_JAVA_OPTS+=" -Dalluxio.logger.type=Console"
+    runJavaClass "$@"
+  ;;
```

Comment:
Maybe we can combine all `*DocGen` functions into a single command? Are there any downsides you see?

I feel like having so many just clutters the interface. They all take such a short amount of time I don't think it would hurt to simply run everything?

Reply:
Yeah, we could do that. I previously thought about combining them. The reason I didn't do that is .... (I don't want to touch ConfigurationDocGen doc which ... needs some time to reformat codes)

Reply:
Changedd to use the same docGen for both metrics and configuration